qa/L0_client/grpc_client.cc

```diff
@@ -0,0 +1,17 @@
+#include "grpc_client.h"
```

Comment:
Put the NV copyright on this file. You can copy it from any source file, use 2020 as year.